app/controllers/AdditionalMetadataNavigationController.scala

```diff
@@ -59,11 +60,4 @@ class AdditionalMetadataNavigationController @Inject() (
       Future(Redirect(routes.AddAdditionalMetadataController.addAdditionalMetadata(consignmentId, metadataType, fileIds)))
     }
   }
-
-  private def getCachedFiles(consignmentId: UUID, metadataType: String, request: Request[AnyContent]): Future[ConsignmentService.File] = {
```

Comment:
Is there a reason for removing the caching?

Reply:
Yeah, we had to remove this so that we can get the latest file status on the page load. I understand that we need to call again in case of an error (the user submits that page without selecting a file) but this is a very rare case.